### DISCUSSION TITLE
How to use "simple" style seen in demo?

### DISCUSSION THREAD

**Discussion by christoferd** [tailwindlabs/tailwindcss#12188] [Help] [answered]
How to use "simple" style seen in demo?

https://tailwindcss-forms.vercel.app/

Maybe I'm understanding this wrong. I assume the form plugin is to add some preset styles/css. Right?

**Comment by wongjn** [comment, marked as answer]
The form plugin gives the *Unstyled* style. To have the *Simple* style, you can copy the class names used, either by inspecting the elements with your browser's devtools or by looking [at the source code](https://github.com/tailwindlabs/tailwindcss-forms/blob/ad3f0564c010bf94cef9f352350a7518729b4102/index.html#L71-L121).
